lib/rules/lines-around-comment.js

```diff
@@ -115,6 +125,11 @@ module.exports = function(context) {
             after = false;
         }
 
+        // if block start is allowed and comment is at block start then return
+        if (options.allowBlockStart && isCommentAtBlockStart()) {
```

Comment:
I don't ever require blank lines after comments, so I'd rather defer to those who might actually use this check. If this is the behavior we go with, then part of the docs would need to be updated:

> This option can be useful when combined with options `beforeLineComment` and `beforeBlockComment` only.